docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+* `Component.with_collection` supports components that accept keyword arguments.
```

Comment:
Can this be changed to "accept splatted keyword arguments"?  I believe defined keyword args are not a problem.

Reply:
That's much clearer!